docs/reference/src/language/index.rst

```diff
@@ -38,5 +40,7 @@ The Slint language describes extensible graphical user interfaces using the
 - Build highly customized user interfaces with the :ref:`builtin elements <Builtin Elements>`
   and pre-built :ref:`widgets <Widgets>` provided.
 
-It only describes the user interface and it is not a programming language. The business
+.. Again, repitition
```

Comment:
Feel free to remove it if you feel it's too repetitive :)

(will the `.. ` be visible in rendering?)

Reply:
It's an rst comment, so no, but I'll just act on it and come back to reducing overall repetition at a later date